src/components/foodDescription/FoodList.js

```diff
@@ -5,15 +5,16 @@ import './FoodList.css'
 
 function FoodList(props){
 
-    const { foodListData } = props;
+    const { foodListData, handleAddToCart } = props;
 
     let keys = Object.keys(foodListData);
```

Comment:
use const and keep the rendering logic(if it's not a simple one) in a helper function above the functional component